docs/marketplace/fastlane/testing-distribution/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Setting Up Appcircle Testing Distribution Plugin
+sidebar_label: Testing Distribution
+description: Enhance powerful action to distribute your builds to appcircle
+tags:
+  [
+    testing-distribution,
+    ipa distribution,
+    apk distribution,
+    binary distribution,
+  ]
+sidebar_position: 1
+---
+
+The Appcircle Testing Distribution plugin allows users to upload their apps and start distribution to test groups or individuals.
+
+### Discover Action
+
+You can discover more about this action and install it by:
+https://rubygems.org/gems/fastlane-plugin-appcircle_testing_distribution
```

Comment:
You can discover more about this action and install it by visiting [here.](https://rubygems.org/gems/fastlane-plugin-appcircle_testing_distribution)